documentation/modules/oauth/con-oauth-authorization-keycloak-example.adoc

```diff
@@ -0,0 +1,458 @@
+// Module included in the following module:
+//
+// con-oauth-authorization-keycloak-authorization-services.adoc
+
+[id='con-oauth-authorization-keycloak-example_{context}']
+= Keycloak Authorization Services Example
+
+This is an end-to-end example of using Keycloak Authorization Services to configure authorization rules for use with `keycloak` authorization in Strimzi.
+The example starts by deploying the Keycloak server with pre-configured realm, requiring no additional configuration.
+We deploy the Kafka cluster configured to use one of the pre-configured realms.
+We connect to the Keycloak Admin Console and demonstrate the sections of the UI where authorizations rules are configured.
+We use Kafka CLI client tools with different personal and service accounts to demonstrate limited access based on the permissions granted to different accounts.
+
+
+## Token Based Authorization with Keycloak Authorization Services
+
+Once the Kafka Broker has obtained an access token by using Strimzi Kafka OAuth for authentication, it is possible to use centrally managed authorization rules to enforce access restrictions onto Kafka Clients.
```

Comment:
```suggestion
Once the Kafka Broker has obtained an access token by using [Strimzi Kafka OAuth](https://github.com/strimzi/strimzi-kafka-oauth) for authentication, it is possible to use centrally managed authorization rules to enforce access restrictions onto Kafka Clients.
```

Reply:
Here I refer to the project ... I made it more clear by turning it into a web link.